plsql/PlSqlParser.g4

```diff
@@ -4294,16 +4303,16 @@ keep_clause
     ;
 
 function_argument
-    : '(' (','? argument)* ')' keep_clause?
+    : '(' argument? (',' argument)* ')' keep_clause?
```

Comment:
@KvanTTT   I fixed this in a wrong way, `(','? argument)*` must be replaced with `(xxx (',' xxx)*)?`